sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -2125,6 +2324,65 @@ node nodeA = myGraph.add([&](handler& cgh) {
 dynParamAccessor.update(bufferB.get_access());
 ----
 
+=== Dynamic Event Update
+
+Example which shows how to create a dependency between two graphs using dynamic
+events which are then updated between executions.
+
+[source,c++]
+----
+...
+
+using namespace sycl;
+namespace sycl_ext = sycl::ext::oneapi::experimental;
+
+queue myQueue;
+auto myContext = myQueue.get_context();
+auto myDevice = myQueue.get_device();
+
+// Create the first graph
+sycl_ext::command_graph graphA(myContext, myDevice);
+
+// Add a node to graphA
+sycl_ext::node nodeA = graphA.add(...);
+
+sycl_ext::command_graph<sycl_ext::graph_state::executable> execGraphA = 
+  graphA.finalize();
+
+// Get an event for the next execution of nodeA in execGraphA
+event eventA = execGraph.get_event(nodeA);
+
+// Create the second graph
+sycl_ext::command_graph graphB(myContext, myDevice);
+
+// Create the dynamic event which will wrap eventA and use it as a dependency
+// for a node in graphB
+sycl_ext::dynamic_event dynEvent{eventA};
+
+auto nodeB = graphB.add([&](handler& CGH){
+  CGH.depends_on(dynEvent);
```

Comment:
I don't think we've added an entry to our table of new `handler` methods an entry for `handler::depends_on(sycl_ext::dynamic_event)`.

I only see `depends_on` extended for `sycl_ext::property::node::depends_on`